dev-lang/moarvm/moarvm-2022.12.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Why EAPI 7?

Reply:
I didn't think much about it. I just inherited it from older ebuilds. I set it to 8 now.